PSSlack/Public/Get-SlackReminder.ps1

```diff
@@ -0,0 +1,94 @@
+function Get-SlackReminder
+{
+    <#
+    .SYNOPSIS
+        Gets a list Slack reminders for the current user or information about
+        the specified Slack reminder using the Id parameter.
+
+    .DESCRIPTION
+        Gets a list Slack reminders for the current user or information about
+        the specified Slack reminder using the Id parameter.
+
+    .PARAMETER Token
+        Token to use for the Slack API
+
+        Default value is the value set by Set-PSSlackConfig
+
+        This takes precedence over Uri
+
+    .PARAMETER Proxy
+        Proxy server to use
+
+        Default value is the value set by Set-PSSlackConfig
+
+    .PARAMETER ReminderId
+        The ID of the reminder
+
+    .EXAMPLE
+        # This is a simple example on how to get a list of all reminders
+
+        Get-SlackReminder
+
+        # Gets a list of reminders for the current user based off of the Token
+
+    .EXAMPLE
+        # This is a simple example on how to get information on a specific reminder
+
+        Get-SlackReminder -Id "RmAZPB1FBP"
+
+        # Gets the information for the reminder specified by the Reminder parameter
+
+    .LINK
+        https://api.slack.com/methods/reminders.info
+
+    .LINK
+        https://api.slack.com/methods/reminders.list
+
+    .FUNCTIONALITY
+        Slack
+    #>
+
+    param (
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [string]$Token = $Script:PSSlack.Token,
+        [string]$ReminderId
+    )
+    begin
+    {
+        $ProxyParam = @{}
+        if ($Proxy)
+        {
+            $ProxyParam.Proxy = $Proxy
+        }
+    }
+    process
+    {
+        $body = @{ }
+
+        if ($ReminderId)
+        {
+            $body.reminder = $ReminderId
+            $method = "reminders.info"
+        }
+        else
+        {
+            $method = "reminders.list"
+        }
+    }
+    end
+    {
+        Write-Verbose "Send-SlackApi"
+        if ($ReminderId)
+        {
+            $response = Send-SlackApi @ProxyParam -Method $method -Body $body -Token $Token -ForceVerbose:$ForceVerbose
+            $returnObj = $response.reminder
+        }
+        else
+        {
+            $response = Send-SlackApi @ProxyParam -Method $method -Token $Token -ForceVerbose:$ForceVerbose
+            $returnObj = $response.reminders
+        }
+        Add-ObjectDetail -InputObject $returnObj -TypeName 'PSSlack.Reminder'
```

Comment:
It might be worth adding a little logic to parse out the time into a `datetime`.  It adds an extra hop, but it can be handy.

E.g. with `Get-SlackFile`, we call `Parse-SlackFile` which includes [this bit](https://github.com/RamblingCookieMonster/PSSlack/blob/master/PSSlack/Private/Parse-SlackFile.ps1#L28).  This also lets you add a friendly `UserName` field, [e.g.](https://github.com/RamblingCookieMonster/PSSlack/blob/master/PSSlack/Private/Parse-SlackFile.ps1#L19)

But... This works, going to just merge it.  I'll add these in separately, sort of owe you after leaving this lingering so long!